src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CSharpCompilerExtensionsResources.resx

```diff
@@ -136,8 +137,8 @@
     <value>'using' directive preferences</value>
     <comment>{Locked="using"} "using" is a C# keyword and should not be localized.</comment>
   </data>
-  <data name="Expected_string_or_char_literal" xml:space="preserve">
-    <value>Expected string or char literal</value>
+  <data name="Expected_string_or_character_literal" xml:space="preserve">
+    <value>Expected string or character literal</value>
```

Comment:
Didn't want to lock 'char' instead?

Reply:
We have the option of locking both `string` and `char`, or going with the approach here. This one seemed readable and lines up [with the specification](https://github.com/dotnet/csharplang/blob/master/spec/lexical-structure.md#character-literals).